src/ErrorBoundary.md

```diff
@@ -0,0 +1,52 @@
+Suspense when delay 3sec:
+
+```js
+import React, { Suspense } from 'react';
```

Comment:
1. I'm unfamiliar with `cypress`. Do we need this file?
2. If we do, let's clean up the indentation please 🙂

Reply:
That sample block like a file, so import is for here

Reply:
Let's just clean up the indentation then to 2 spaces 🙂